packages/editor/src/components/post-card-panel/index.js

```diff
@@ -106,3 +130,59 @@ export default function PostCardPanel( { className, actions } ) {
 		</PanelBody>
 	);
 }
+
+// Taken from packages/editor/src/components/time-to-read/index.js.
+const AVERAGE_READING_RATE = 189;
+
+// This component renders the wordcount and reading time for the post.
+function PostContentInfo( { postContent } ) {
+	/*
+	 * translators: If your word count is based on single characters (e.g. East Asian characters),
+	 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
+	 * Do not translate into your own language.
+	 */
+	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
+	const wordsCounted = postContent
+		? wordCount( postContent, wordCountType )
+		: 0;
+	if ( ! wordsCounted ) {
+		return null;
+	}
+	const readingTime = Math.round( wordsCounted / AVERAGE_READING_RATE );
+	let wordsCountText;
+	if ( ! wordsCounted.toLocaleString() ) {
+		wordsCountText = __( 'Unknown' );
```

Comment:
* What's the point of displaying 'Unknown words' to the user? I'd remove it.
* If we **still** want to display that, this should be a clearer phrase, e.g. "Unknown words", and come with a `translators` note
* Could be a `const` with a ternary expression :P

Reply:
Updated here: https://github.com/WordPress/gutenberg/pull/60672/files#diff-1e4f9f78eef092c02d177ba9df94fa6f74437139cdefe8b85b4863840555fe28R171